Cargo.toml

```diff
@@ -8,20 +8,20 @@ repository = "https://github.com/paritytech/finality-grandpa"
 edition = "2018"
 
 [dependencies]
-futures = "0.1"
+futures = "0.3.1"
+futures-timer = "2.0.2"
```

Comment:
does it still compile in `no-std`? I would have thought we needed `no-default-features` on these futures crates.

Reply:
Fixed.